deny.toml

```diff
@@ -49,6 +49,8 @@ skip = [
     { name = "ndk-sys", version = "0.2" },              # from winit v0.26.1
     { name = "stdweb", version = "0.1" },               # from rodio v0.15.0
     { name = "wasi",  version = "0.10"},                # from ahash v0.7.6
+    { name = "nix", version = "0.23.1" },               # from alsa v0.6.0
+    { name = "nix", version = "0.24.1" },               # from gilrs-core v0.4.0
```

Comment:
```suggestion
```
no need to mention the two versions, that way we'll be notified if something start depending on a third version of `nix`